docs/sdk/core-kit/mpc-core-kit/usage.mdx

```diff
@@ -10,11 +10,12 @@ provider given by Web3Auth to sign transactions and interact with the blockchain
 
 Natively, the instance of `Web3AuthMPCCoreKit` (referred to as `coreKitInstance` in our examples) returns the following functions:
 
-Authentication Functions:
+Authentication and Handling Functions:
 
-- `loginWithOAuth()` - Logs in the user in OAuth Intrisic Flow.
-- `loginWithJWT()` - Logs in the user with a JWT based ID Token.
+- `loginWithOAuth()` - Logs in the user in OAuth Intrinsic Flow.
```

Comment:
intrinsic or implicit?

Reply:
Intrinsic.

It was a typo from earlier. I didn't check. Thanks for bringing this to my attention. Updated at all places.